### DISCUSSION TITLE
Want to pass api key through some token inside GrapesJS

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5423] [Q&A] [answered]
Hello, I wanted to pass the apiKey of an SDK through some token is it possible to do that inside GrapesJS.

My Code look like this.

      const scriptElement = document.createElement("script"); 
       scriptElement.src = 
         "https://unpkg.com/sfd-sdk-javascript-grapejs@1.0.36/dist/index.min.js"; 
       scriptElement.onload = async () => { 
         // Logic for SDK initialization after script load 
         func1(); 
         await Initialize_SDK({ 
           ProjectApiKey: "v201P.tf5MatajUsMSvw0zu", 
           ApiURL: "https://zoom-sf.genderseparator.com/", 
         });

I want to pass the ProjectKey as a token right now I'm hard coding it.

**Comment by artf** [comment, marked as answer]
It's explained here how to pass properties to scripts https://grapesjs.com/docs/modules/Components-js.html#passing-properties-to-scripts
